web/controllers/post_controller.ex

```diff
@@ -25,7 +25,7 @@ defmodule Healthlocker.PostController do
   end
 
   def index(conn, _params) do
-    posts = Repo.all(Post)
+    posts = Repo.all(from p in Post, where: ilike(p.content, "%#story%"))
```

Comment:
It'd be nice to move this query into the model, something like this: https://blog.drewolson.org/composable-queries-ecto/

Then we could test it independently there.

Also we should probably add a controller test, not sure if I finished those before.